common/build/build-common/api-extractor-base.json

```diff
@@ -156,6 +156,12 @@
 				"addToApiReportFile": false
 			},
 
+			// Require documentation on all package-exported API items.
+			"ae-undocumented": {
+				"logLevel": "error",
+				"addToApiReportFile": true // TODO: promote this to `false` once violations in consumers have been addressed.
```

Comment:
could we flip this, and set it to false by default, and then out-opt any back packages. the cycle to update this is a bit long with how we depend on it. that strategy will also prevent more being added.

Reply:
We can, though I expect that will be most packages. We'll also need to make the version bump here a major one, rather than a patch.

Reply:
ok. either way then. especially if we can do this cheaply in a patch, then follow with a major